cpp/src/io/statistics/statistics_type_identification.cuh

```diff
@@ -123,8 +125,11 @@ class extrema_type {
 
   using non_arithmetic_extrema_type = typename std::conditional_t<
     cudf::is_fixed_point<T>() or cudf::is_duration<T>() or cudf::is_timestamp<T>(),
-    typename std::conditional_t<std::is_same_v<T, numeric::decimal128>, __int128_t, int64_t>,
-    typename std::conditional_t<std::is_same_v<T, string_view>, string_view, void>>;
+    int64_t,
```

Comment:
This reverts #11179 :(

Reply:
Yeah, I spotted that at the end of last night when the new tests failed. I'll get it added back.

Reply:
Do you ever sleep?

Reply:
I plug my toes in at night, does that count?